cmd/editorconfig-checker/main.go

```
@@ -64,6 +65,10 @@ func init() {
 
 	currentConfig, _ = config.NewConfig(configPaths)
```

Side note, out of this PR consideration

@theoludwig @klaernie why are we ignoring the error here?

---

Same question for `config.Parse()` call a few liner below

---

I've asked myself the same question for #380 and rewrote a bit of it, didn't I? But no matter, that should be a separate issue.

---

Yes, exactly

---

Just read over it again, to be sure: in #380 I started verifying the error returned by `config.Parse()`, since that is the relevant point where reading a custom config file must succeed for us to continue, yet we can safely ignore the error if we tried to parse the default configuration file.